### PR TITLE
Missing link for Vertical-scaling in bare_metal_machine.md

### PR DESCRIPTION
**Pull request by dipankardas011**
Signed-off-by: Dipankar Das <dipsonu10@hotmail.com>

### CODE REVIEW

**Review by CathPag — APPROVED** [review]
Great catch!

**Comment by CathPag** [comment]
@jihoon-seo, do you know why these checks are failing? It's only a new link

File: content/en/bare_metal_machine.md [hunk 1]
```diff
@@ -16,4 +16,4 @@ Pairing one operating system with one physical computer is the original pattern
...
+In the context of [cloud native apps](/cloud_native_apps/), we generally think of performance in terms of [scaling](/scalability/) to a large number of concurrent events, which can be handled by [horizontal scaling](/horizontal_scaling/) (adding more machines to your resource pool). But some workloads may require [vertical scaling](/vertical_scaling/) (adding more power to an existing physical machine) and/or an extremely fast physical hardware response in which case bare metal is better suited. Bare metal also allows you to tune the physical hardware and possibly even hardware drivers to help accomplish your task.
```
Comment (jihoon-seo) [inline content/en/bare_metal_machine.md:19]: Unfortunately, closing and reopening this PR did not triggered Netlify build. 😥
@dipankardas011 Could you push another commit that changes nothing in actual, like adding empty line in the end of the file?

```suggestion
In the context of [cloud native apps](/cloud_native_apps/), we generally think of performance in terms of [scaling](/scalability/) to a large number of concurrent events, which can be handled by [horizontal scaling](/horizontal_scaling/) (adding more machines to your resource pool). But some workloads may require [vertical scaling](/vertical_scaling/) (adding more power to an existing physical machine) and/or an extremely fast physical hardware response in which case bare metal is better suited. Bare metal also allows you to tune the physical hardware and possibly even hardware drivers to help accomplish your task.

```

Comment (dipankardas011) [reply content/en/bare_metal_machine.md:19]: okay pushing a commit🙂

**Comment by CathPag** [comment]
> @CathPag I guess that there was a problem with Netlify for a while. I am going to close and reopen this PR to trigger another Netlify build. 😊

Thank you!!

**Review by jihoon-seo — APPROVED** [review]

**Pull request merged**